index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube Player</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <!--youtube sample webpage-->
+    <header>
+        <img class="logo-nav-bar" src="./images/ytlogo.jpg" alt="">
+        <nav class="navbar">
+            <div class="nav-bar-cascade">
+                <ul>
+                    <il><a href="#">Home</a></il>
+                    <il><a href="#">Shorts</a></il>
+                    <il><a href="#">Suscriptions</a></il>
+                    <il><a href="#">Library</a></il>
+                </ul>
+            </div> 
+            <div class="nav-bar-search-bar">
+                <input type="text" placeholder="Search">
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="player-container">
+            <iframe class="player" width="560" height="315" src="https://www.youtube.com/embed/lM02vNMRRB0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
- [ ] I notice you properly used a class inside the tag but I kindly recommend checking the inline styles, it is always a good practice to keep your style definitions in the style sheet, for instance:

```
.player {
  width: 560px;
  height: 315px;
}
```